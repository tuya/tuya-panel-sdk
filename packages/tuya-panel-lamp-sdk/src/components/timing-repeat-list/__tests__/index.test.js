import React from 'react';
import { shallow } from 'enzyme';
import TimingRepeatList from '../index';

describe('TimingRepeatList components', () => {
  it('basic render', () => {
    const wrapper = shallow(<TimingRepeatList />);
    expect(wrapper).toMatchSnapshot();
  });
});
