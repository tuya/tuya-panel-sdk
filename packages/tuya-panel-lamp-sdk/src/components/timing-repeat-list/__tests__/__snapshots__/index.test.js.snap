// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimingRepeatList components basic render 1`] = `
<ScrollViewMock
  contentContainerStyle={
    Array [
      Object {
        "paddingBottom": 64,
      },
      Object {},
    ]
  }
  style={
    Array [
      Object {
        "maxHeight": 298,
        "overflow": "hidden",
        "paddingHorizontal": 32,
        "width": "100%",
      },
      Object {},
    ]
  }
>
  <TouchableOpacity
    activeOpacity={1}
  >
    <View
      key="0"
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#F5F5F5",
            "borderRadius": 54,
            "flexDirection": "row",
            "height": 108,
            "justifyContent": "space-between",
            "marginTop": 18,
            "paddingHorizontal": 48,
            "width": "100%",
          },
          Object {},
        ]
      }
    >
      <SingleTime
        is24Hour={true}
        time={720}
      />
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "rgba(16,130,254,0.1)",
              "borderRadius": 8,
              "justifyContent": "center",
              "marginLeft": 32,
              "paddingHorizontal": 8,
              "paddingVertical": 4,
            },
            Object {},
          ]
        }
      >
        <TYText
          align={null}
          color={null}
          size={null}
          style={
            Array [
              Object {
                "color": "#39a9ff",
                "fontSize": 24,
              },
              Object {},
            ]
          }
          type={null}
          weight={null}
        >
          Normal
        </TYText>
      </View>
    </View>
  </TouchableOpacity>
</ScrollViewMock>
`;
